### DISCUSSION TITLE
CIS Microsoft Edge Benchmark v3.0.0

### DISCUSSION THREAD

**Discussion by Smokuuu** [0x6d69636b/windows_hardening#171] [Ideas]
Hello everyone,

First of all thank you for developing that tool! We love it here :)

My question is, if it is possible for you to add "CIS Microsoft Edge v3.0.0" as a benchmark list.
It is much more detailed than the Microsoft's ones and cover much more stuff.
I think it would be fine addition to the collection.

Thank you!

**Comment by 0x6d69636b** [comment]
I can add it to my to-do list. Convert CIS Benchmark is tedious work to "extract" the recommendations from the PDF into my database
